src/index.js

```diff
@@ -0,0 +1,36 @@
+import './styles.css';
+import display from './modules/display.js';
+import { getAPI, setAPI, createGame } from './modules/API.js';
+
+const initializeGame = async () => {
+  console.log(await createGame('roblox'));
+
+  const getScore = async () => {
+    const scores = await getAPI();
+    display(scores);
+  };
+  const form = document.querySelector('form');
+
+  form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const name = document.querySelector('.name');
+    const score = document.querySelector('.score');
+    const playername = name.value.trim();
+    const playerscore = score.value.trim();
+    const data = {
+      user: playername,
+      score: playerscore,
+    };
+    name.value = '';
+    score.value = '';
+    await setAPI(data);
+  });
+
+  document.querySelector('.refresh').addEventListener('click', () => {
+    getScore();
+  });
+
+  getScore();
+};
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍